README.md

```diff
@@ -6,9 +6,10 @@ This is a prototype debugger written without any XUL and based on React and Redu
 
 #### Getting Started
 
-```js
+```
 $ npm install
 $ npm start
+$ python -m SimpleHTTPServer 8000 
```

Comment:
This is no longer necessary: [npm start](https://github.com/jlongster/debugger.html/blob/master/package.json#L4)